src/go/rpk/pkg/cli/cmd/cluster/config/import.go

```diff
@@ -195,6 +195,14 @@ func importConfig(
 
 	fmt.Printf("Successfully updated configuration. New configuration version is %d.\n", result.ConfigVersion)
 
+	status, err := client.ClusterConfigStatus(ctx, true)
+	out.MaybeDie(err, "error fetching status: %v", err)
+	for _, value := range status {
+		if value.Restart {
+			fmt.Print("\nCluster needs to be restarted. See more details with 'rpk cluster config status'.\n")
```

Comment:
I think we should return when we find the first `value.Restart`, otherwise we could end up with multiple lines of :

```
Successfully updated configuration. New configuration version is 3

Cluster needs to be restarted. See more details with 'rpk cluster config status'.

Cluster needs to be restarted. See more details with 'rpk cluster config status'.

Cluster needs to be restarted. See more details with 'rpk cluster config status'.
```

Same in config set.

Reply:
Great catch, thank you!